windows/src/developer/inst/copydev.in

```diff
@@ -1,5 +1,10 @@
 !include ..\..\Defines.mak
 
+!ifndef TIER
+    # In dev environments, we'll hack the tier to stable; CI sets this for us.
+TIER=alpha
```

Comment:
The comment says `hack the tier to stable` but the tier is set to alpha? Please make these consistent.